src/transactions/feed/detailContent/SwapContent.tsx

```diff
@@ -0,0 +1,133 @@
+import BigNumber from 'bignumber.js'
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import TokenDisplay, { formatValueToDisplay } from 'src/components/TokenDisplay'
+import Touchable from 'src/components/Touchable'
+import OpenLinkIcon from 'src/icons/OpenLinkIcon'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { tokensListSelector } from 'src/tokens/selectors'
+import { TokenExchange } from 'src/transactions/types'
+import networkConfig from 'src/web3/networkConfig'
+
+export interface Props {
+  exchange: TokenExchange
+}
+
+// Note that this is tested from TransactionDetailsScreen.test.tsx
+export default function SwapContent({ exchange }: Props) {
+  const { t } = useTranslation()
+  const tokensList = useSelector(tokensListSelector)
+
+  const fromTokenSymbol = tokensList.find(
+    (token) => token.address === exchange.outAmount.tokenAddress
+  )?.symbol
+  const toTokenSymbol = tokensList.find((token) => token.address === exchange.inAmount.tokenAddress)
+    ?.symbol
+
+  //TODO: verifying fee part.
+  const onPressTxDetails = () => {
+    navigate(Screens.WebViewScreen, {
+      uri: `${networkConfig.celoExplorerBaseTxUrl}${exchange.transactionHash}`,
+    })
+  }
+
+  return (
+    <View style={styles.contentContainer}>
+      <View style={[styles.row, { paddingBottom: Spacing.Regular16 }]}>
+        <Text style={styles.exchangeBodyText}>{t('swapContent.swapTo')}</Text>
+        <TokenDisplay
+          style={styles.currencyAmountText}
+          amount={exchange.outAmount.value}
+          tokenAddress={exchange.outAmount.tokenAddress}
+          showLocalAmount={false}
+          showSymbol={true}
+          hideSign={true}
+          testID="SwapContent/swapFrom"
+        />
+      </View>
+      <View style={[styles.row, { paddingBottom: Spacing.Regular16 }]}>
+        <Text style={styles.exchangeBodyText}>{t('swapContent.swapFrom')}</Text>
+        <TokenDisplay
+          style={styles.currencyAmountText}
+          amount={exchange.inAmount.value}
+          tokenAddress={exchange.inAmount.tokenAddress}
+          showLocalAmount={false}
+          showSymbol={true}
+          showExplicitPositiveSign={true}
+          hideSign={true}
+          testID="SwapContent/swapTo"
+        />
+      </View>
+      <View style={styles.separator} />
+      <View style={[styles.row, { paddingBottom: Spacing.Smallest8 }]}>
```

Comment:
can we just have `row` use one `paddingBottom` value? not sure i see the point of 8 vs 16, we can make it 12 for all if we want?

otherwise would advise to have `row` and `rowWithSpacing` rather than have inline styles like this

Reply:
The mocks have a space difference between the rows from what I was seeing 16 on the top two and 8 on the bottom ones.

Reply:
right but it's a bit of give an take with design - if changing a few pixels doesn't compromise the meaning of the design and can reduce complexity in the code, i think it's worth doing. it's too late now because Nitya is on holiday so we can keep it like this, but i do think this is the mentality we should carry going forward. it's in all of our best interest to keep our code (and styles) as simple as possible, when designing it's not always obvious what would cause code complexity and sometimes those parts of the design aren't important. so if we double check with the designer, at least we can be sure that any complexity that we do introduce is worth the effort